cps/src/main/java/org/jenkinsci/plugins/workflow/cps/nodes/StepEndNode.java

```diff
@@ -38,6 +38,17 @@ protected String getTypeDisplayName() {
     }
 
     @Override
+    protected String getTypeFunctionName() {
+        StepDescriptor d = getDescriptor();
+        boolean isBody = getStartNode().isBody();
+        if (isBody) {
+            return "} //" + (d != null ? d.getFunctionName() : getStartNode().getStepName());
+        } else {
+            return getStartNode().getStepName() + " : End";
```

Comment:
I assume there's no overhead to `getStartNode`? It's called 3 times here in this method. I only ask because this flow node stuff is mad and it would not surprise me if `getStartNode` involved some sort of convoluted graph traversal.